hw1/02-palindrome.js

```diff
@@ -1,3 +1,46 @@
-const elem = document.querySelector('input');
+const inputElem = document.querySelector("input");
+const NOT_A_NUMBER = "Not a number.";
+const POSITIVES_ONLY = "Input cannot be less than 0.";
+const NOT_PALINDROME = "No. Try again";
+const IS_PALINDROME = "Yes! This number is a palindrome!";
 
-elem.addEventListener('input', handleInput);
+inputElem.addEventListener("input", handleInput);
+
+function inputVerify(input) {
+    const verifyResult = isNaN(input) === true || input.length === 0;
+    return verifyResult;
+}
+
```

Comment:
It's not a huge program so this is less important, but adding some comments to your functions would be beneficial.